cmd/machine-config-daemon/pivot.go

```diff
@@ -291,10 +297,12 @@ func pullAndRebase(container string) (imgid string, changed bool) {
 			glog.Infof("Using ref %s", refs[0])
 			ostreeCsum = utils.RunGetOut("ostree", "rev-parse", "--repo", repo, refs[0])
 		} else if len(refs) > 1 {
-			glog.Fatalf("Multiple refs found in repo!")
+			err = errors.New("multiple refs found in repo")
+			return
 		} else {
 			// XXX: in the future, possibly scan the repo to find a unique .commit object
-			glog.Fatalf("No refs found in repo!")
+			err = errors.New("No refs found in repo")
```

Comment:
why are we setting err here and above?

Reply:
Because we're using the named return value.